src/lib/decorators.d.ts

```diff
@@ -38,27 +49,39 @@ interface ClassDecoratorContext<Class extends abstract new (...args: any) => any
     addInitializer(initializer: (this: Class) => void): void;
 }
 
-// #region type ClassDecoratorFunction
-
 /**
- * Describes a function that can be used to decorate a class.
+ * Describes the call signature a generic function that can be used to decorate a class.

```

Comment:
There is a missing word here or something, but I'm not sure it's 'of':

```suggestion
 * Describes the call signature of a generic function that can be used to decorate a class.
```